pkg/store/verifiable/store.go

```diff
@@ -69,3 +86,13 @@ func (s *Store) GetVC(vcID string) (*verifiable.Credential, error) {
 
 	return vc, nil
 }
+
+// GetCredentialIDBasedOnName retrieves verifiable credential id based on name.
+func (s *Store) GetCredentialIDBasedOnName(name string) (string, error) {
```

Comment:
GetCredentialD`ByName` maybe? `BasedOnName` seems odd.

or no suffix at all since the `name` argument implies it's going to be filtered by `name`

Reply:
[Updated](https://github.com/hyperledger/aries-framework-go/compare/c6d66bb04d814caeb92a753132b79a8818daa699..cc1f4e1f94378cce223bd1aa5655a9922e50e3fd#diff-94f07b1b8913fb6c9dbe128104e469c9R91).